generators/languages/templates/src/main/webapp/i18n/en/global.json.ejs

```diff
@@ -152,7 +152,8 @@
         "Size": "Field {{ fieldName }} does not meet min/max size requirements!",
         "userexists": "Login name already used!",
         "emailexists": "Email is already in use!",
-        "idexists": "A new {{ entityName }} cannot already have an ID"
+        "idexists": "A new {{ entityName }} cannot already have an ID",
+        "idnull": "Invalid ID"
```

Comment:
Please add the keybto all other languages as well. No need to translate just add the English version

Reply:
@deepu105 I have added to remaining langs, please review and merge.